### DISCUSSION TITLE
Format of dataset

### DISCUSSION THREAD

**Discussion by tiomaldy** [coqui-ai/TTS#668] [General Q&A]
The datasets need a metada_mls  json   archive ?
![image](https://user-images.githubusercontent.com/61519589/126109643-28925c15-fb29-46cd-bfa5-b61b9655c13f.png)
Because i make my own dataset but i don t make that archive its just like this 
![image](https://user-images.githubusercontent.com/61519589/126109692-2582dcce-ad1c-4914-871f-ad8c4544b358.png)

**Comment by erogol** [comment]
https://tts.readthedocs.io/en/latest/formatting_your_dataset.html
